react-main/src/app/components/NavBar/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+import NavBarLinks from './components/NavBarLinks';
+import Logout from './components/Logout';
+
+class NavBar extends Component {
+  render() {
+    return (
+      <div className="navbar">
+        <NavBarLinks />
+        <Logout {...this.props} />
```

Comment:
Define which props you need to send. This way you are not defining any proptypes.